packages/@lwc/engine-server/src/__tests__/fixtures/attribute-aria-modify/expected.html

```diff
@@ -1,4 +1,3 @@
 <x-cmp aria-busy="true" aria-activedescendant="foo">
-  <template shadowroot="open">
-  </template>
-</x-cmp>
\ No newline at end of file
+  <template shadowroot="open"></template>
+</x-cmp>
```

Comment:
Switching to prettier means I am modifying a bazillion fixture files, but the only new ones are `styles-escaping-unsafe` and `styles-escaping-safe`.